### PR TITLE
media: Modify SpeechDetector deinit return value

### PR DESCRIPTION
**Pull request by junmin-kim**
SpeechDetector::deinitXXX returns false if it is already deinit

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [1898adc678406f44de0e3d63608cf212bdf934bc] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [1898adc678406f44de0e3d63608cf212bdf934bc] - Code Rule Check Result:
framework/include/media/voice/SpeechDetector.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/voice/SpeechDetector.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/voice/SpeechDetector.h:38: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/voice/SpeechDetector.h:38: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/voice/SpeechDetector.h:39: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/voice/SpeechDetector.h:39: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/voice/SpeechDetector.h:39: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/voice/SpeechDetector.h:39: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)

File: apps/examples/testcase/ta_tc/media/utc/utc_media_speechdetector.cpp [hunk 1]
```diff
@@ -64,11 +64,35 @@ static void utc_media_SpeechDetector_initEndPointDetect_n(void)
 
 static void utc_media_SpeechDetector_deinitKeywordDetect_p(void)
 {
+	auto instance = media::voice::SpeechDetector::instance();
+	instance->initKeywordDetect(16000, 2);
```
Comment (thapav) [inline apps/examples/testcase/ta_tc/media/utc/utc_media_speechdetector.cpp]: May be a #define would be more suitable for the values used
#define TEST_SAMPLE_RATE 16000
#define TEST_CHANNELS 2

Comment (junmin-kim) [reply apps/examples/testcase/ta_tc/media/utc/utc_media_speechdetector.cpp]: Thanks fro the review. I'll fix it

**Comment by seinfra** [comment]
Target : [77076398d2b862703ef29874d5d401878a09f06b] - Code Rule Check Result:
framework/include/media/voice/SpeechDetector.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/voice/SpeechDetector.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/voice/SpeechDetector.h:38: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/voice/SpeechDetector.h:38: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/voice/SpeechDetector.h:39: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/voice/SpeechDetector.h:39: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/voice/SpeechDetector.h:39: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/voice/SpeechDetector.h:39: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)

**Comment by seinfra** [comment]
Target : [77076398d2b862703ef29874d5d401878a09f06b] - Code Rule Check (C++) OK.

**Pull request merged**